src/anonymous-purchase-consequences-screen/AnonymousPurchaseConsequencesScreenComponent.tsx

```diff
@@ -53,9 +56,16 @@ export const AnonymousPurchaseConsequencesScreenComponent = ({
         onPressLogin ? continueWithoutLoginButton : undefined
       }
     >
-      <ThemeText type="heading--big" color={themeColor} style={styles.header}>
-        {t(AnonymousPurchasesTexts.consequences.title)}
-      </ThemeText>
+      <View ref={focusRef} accessible>
+        <ThemeText
+          accessible
```

Comment:
Should it be `accessible` here? I've experienced on Android that if you have `accessible` inside `accessible`, then you are able to focus both levels.